plugins/kafka/kafka.cpp

```diff
@@ -495,6 +560,7 @@ namespace KafkaPlugin
         consumerPtr = NULL;
         topicPtr = NULL;
 
+#ifndef _CONTAINERIZED
         char cpath[_MAX_DIR];
 
         GetCurrentDirectory(_MAX_DIR, cpath);
```

Comment:
I think you'll get a compiler warning because not checking the return value for this (getpwd).
(we validate return elsewhere)

Reply:
I added a check with an exception if the cwd couldn't be found.